6-pubsub/package.json

```diff
@@ -41,13 +41,14 @@
     "@google/cloud-debug": "^0.8.3",
     "@google/cloud-trace": "^0.5.5",
     "@google-cloud/datastore": "^0.1.1",
+    "@google-cloud/storage": "^0.1.1",
+    "@google-cloud/pubsub": "^0.2.0",
```

Comment:
The latest version of `@google-cloud/pubsub` is `0.3.0`. Are you wanting to upgrade in this PR?